nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -424,15 +433,16 @@ in {
         example = "DE";
         description = lib.mdDoc ''
           ::: {.warning}
-          This option exists since Nextcloud 21! If older versions are used,
-          this will throw an eval-error!
+          This option was introduced in Nextcloud 21! If older versions
+          are used, this will throw an evaluation error!
           :::
```

Comment:
Considering that the minimum version we support nowadays is v25, we can drop this warning entirely.

Reply:
Alright, removed it